unified_targets/configs/TMOTORF7.config

```diff
@@ -0,0 +1,105 @@
+# version
```

Comment:
The line with `# version` needs to be removed.

Reply:
Sure
Maybe we should write a simple script to strip unnecessary stuff for the user. Python would do in my opinion. What do you think?

Reply:
@DieHertz: All for it - we are trying to make creation / submission of (working) Unified Target configurations as easy as possible.